components/community-groups/CommunityGroupsCard.jsx

```diff
@@ -32,6 +32,11 @@ export default function CommunityGroupsCard(props) {
 
   currCG.leadersNamesString = stringifyLeaderNames(currCG.leadersNames);
 
+  const removalText = () => {
+    return ("Are you sure you would like to permanently remove this Community Group " +

```

Comment:
Two things, first the next block of text should still be part of this quote not concatenated separately.

Second, I would change this to 
``` js
`Are you sure you would like to permanently remove this Community Group led by ${currCG.leadersNamesString}?`
```